### PR TITLE
PeopleEatCameraButton

### PR DESCRIPTION
**Pull request by Llavrov**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -25,13 +27,15 @@ const HomePage: NextPage = () => {
...
+                <PeopleEatCameraButton onClick={(): string => 'click'} />
+                <PeopleEatEditButton onClick={(): string => 'click'} />
                 <PeopleEatNextButton />
                 <PeopleEatInput />
                 <PeopleEatInput pass />
                 <PeopleEatInput disabled />
                 <PeopleEatInput email />
                 <PeopleEatCreditCard number={'*1913'} label={'MasterCard'} />
-                <PeopleEatDocumentDownloadButton downloadSize={'245 Mb'} title={'European'} />
+                <PeopleEatDocumentDownloadButton downloadSize={'245 Mb'} title={'European'} onClick={(): string => 'click'} />
```
Comment (CemYil03) [inline src/pages/index.tsx:38]: Actually the return type of the callback is void, not string

File: src/components/standard/nextButton/PeopleEatNextButton.tsx [hunk 2]
```diff
@@ -22,14 +22,16 @@ export default function PeopleEatNextButton({ onClick }: PeopleEatNextButtonProp
...
+                width: '46px',
                 padding: 0,
             }}
             size={'small'}
             variant="contained"
             onClick={handleClick}
-            className={'min-w-[46px] max-w-[46px] min-h-[46px] h-[46px] '}
         >
-            <PeopleEatIcon icon={Icon.arrowRight} className={classNames('', { ['invert']: active })} edgeLength={46} />
+            <PeopleEatIcon icon={Icon.arrowRight} className={classNames('', { ['invert']: active })} edgeLength={24} />
```
Comment (CemYil03) [inline src/components/standard/nextButton/PeopleEatNextButton.tsx:34]: The default for edgeLength already is 24 (see https://github.com/people-eat/web-app/blob/c46016ade369f5eb42cf3c0e73af36755bf9ae1e/src/components/standard/icon/PeopleEatIcon.tsx#L5) so you don't need to set it here explicitly.

File: src/components/standard/hideButton/PeopleEatHideButton.tsx [hunk 3]
```diff
@@ -15,11 +15,11 @@ export default function PeopleEatHideButton({ disabled = false, defaultChecked =
         width: 24,
         height: 24,
         boxShadow: 'boxShadow: 0 0 0 1px transparent',
-        backgroundImage: 'url(/icons/eye-crossed-out.svg)',
+        backgroundImage: 'url(/public/icons/eye-crossed-out.svg)',
     }));
 
     const BpCheckedIcon = styled(BpIcon)({
-        backgroundImage: 'url(/icons/eye.svg)',
+        backgroundImage: 'url(/public/icons/eye.svg)',
```
Comment (CemYil03) [inline src/components/standard/hideButton/PeopleEatHideButton.tsx:22]: I exepect this t not work locally am I right?

File: src/components/standard/icon/PeopleEatIcon.tsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { type PeopleEatIconProps } from './PeopleEatIconProps';
 
 export default function PeopleEatIcon({ className, edgeLength = 24, icon }: PeopleEatIconProps): ReactElement {
     return (
-        <div>
+        <div style={{ height: `${edgeLength}px`, width: `${edgeLength}px` }}>
```
[the remark this answers is not in the document]
Comment (Llavrov) [reply src/components/standard/icon/PeopleEatIcon.tsx:7]: try to change on span - but it doesn't work(

File: src/pages/index.tsx (hunk 1 above)
Comment (CemYil03) [reply src/pages/index.tsx]: What is a stub function? But yes, I think you should change it to void.

File: src/components/standard/hideButton/PeopleEatHideButton.tsx (hunk 3 above)
Comment (CemYil03) [reply src/components/standard/hideButton/PeopleEatHideButton.tsx:23]: Then I would say it is probably not a good solution...

Comment (Llavrov) [reply src/components/standard/hideButton/PeopleEatHideButton.tsx:23]: Okay, just then will search how to use backgroundImage url in this case

**Review by CemYil03 — APPROVED** [review]

**Pull request merged**